vespaclient-container-plugin/src/main/java/com/yahoo/vespa/http/server/ErrorCode.java

```diff
@@ -30,4 +35,16 @@ public boolean isTransient() {
         return _transient;
     }
 
+    static ErrorCode fromBusError(Error mbusError) {
+        Collection<Integer> fatalsHandledAsTransient = Set.of(
```

Comment:
This could probably be moved out to a `private static final` since it is immutable